react/features/aeternity/components/TipButton.js

```diff
@@ -287,7 +288,8 @@ class TipButton extends Component<Props, State> {
         const { t } = this.props;
         const amount = aeternity.util.aeToAtoms(this.state.value);
         const url = `${URLS.SUPER}/user-profile/${this.props.account}`;
-        const { message } = this.state;
+        const DEFAULT_MESSAGE = `Appreciation from conference : ${APP.conference.roomName} on ${window.location.host}.`;
```

Comment:
we don't need that text anymore

Reply:
but if i'm not wrong, you and Yuri said that we need comment while we dont get new version of contract (or something like that)

Reply:
yes, but the field is required, so ppl will have to type something anyway

Reply:
okay, got it